### PR TITLE
Spatial libs

### PR DESCRIPTION
**Pull request by BryanRumsey**

### CODE REVIEW

File: singleuser/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 RUN python3 -m pip install --no-cache \
   jupyterhub==$JUPYTERHUB_VERSION \
-  gillespy2
+  gillespy2 \
+  vtk \
+  https://github.com/spatialpy/SpatialPy/tarball/develop \
+  pygmsh==5.0.2 \
+  meshio==2.3.10 
+
+USER jovyan
```
Comment (briandrawert) [inline singleuser/Dockerfile]: As long as we are editing this file.  Can we rename the user to "stochss"?

Comment (popensesame) [reply singleuser/Dockerfile]: Yup. I think all we need to do is change the value of NB_USER in the base-notebook jupyter image.

The top of singleuser/Dockerfile would be:
```
ARG NB_USER=stochss
ARG DOCKER_NOTEBOOK_IMAGE
FROM $DOCKER_NOTEBOOK_IMAGE
```

And then later change the `USER` directive to `stochss`.

See:
https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile
and
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Comment (Hit2theMo) [reply singleuser/Dockerfile]: Hey, sorry to add a comment here but can someone please check a new issue I just added! Thanks

Comment (BryanRumsey) [reply singleuser/Dockerfile]: Wasn't able to get this change implemented and get the user container to spawn correctly.

**Pull request merged**